src/Module/Admin/Federation.php

```diff
@@ -0,0 +1,198 @@
+<?php
+
+namespace Friendica\Module\Admin;
+
+use Friendica\Core\Config;
+use Friendica\Core\L10n;
+use Friendica\Core\Renderer;
+use Friendica\Database\DBA;
+use Friendica\Module\BaseAdminModule;
+
+class Federation extends BaseAdminModule
+{
+	public static function content()
+	{
+		parent::content();
+
+		// get counts on active friendica, diaspora, redmatrix, hubzilla, gnu
+		// social and statusnet nodes this node is knowing
+		//
+		// We are looking for the following platforms in the DB, "Red" should find
+		// all variants of that platform ID string as the q() function is stripping
+		// off one % two of them are needed in the query
+		// Add more platforms if you like, when one returns 0 known nodes it is not
+		// displayed on the stats page.
+		$platforms = ['Friendi%%a', 'Diaspora', '%%red%%', 'Hubzilla', 'BlaBlaNet', 'GNU Social', 'StatusNet', 'Mastodon', 'Pleroma', 'socialhome', 'ganggo'];
+		$colors = [
+			'Friendi%%a' => '#ffc018', // orange from the logo
+			'Diaspora'   => '#a1a1a1', // logo is black and white, makes a gray
+			'%%red%%'    => '#c50001', // fire red from the logo
+			'Hubzilla'   => '#43488a', // blue from the logo
+			'BlaBlaNet'  => '#3B5998', // blue from the navbar at blablanet-dot-com
+			'GNU Social' => '#a22430', // dark red from the logo
+			'StatusNet'  => '#789240', // the green from the logo (red and blue have already others
+			'Mastodon'   => '#1a9df9', // blue from the Mastodon logo
+			'Pleroma'    => '#E46F0F', // Orange from the text that is used on Pleroma instances
+			'socialhome' => '#52056b', // lilac from the Django Image used at the Socialhome homepage
+			'ganggo'     => '#69d7e2', // from the favicon
+		];
+		$counts = [];
+		$total = 0;
+		$users = 0;
+
+		foreach ($platforms as $platform) {
+			// get a total count for the platform, the name and version of the
+			// highest version and the protocol tpe
+			$platformCountStmt = DBA::p('SELECT

```

Comment:
You could use ```DBA::fetchFirst``` here and include a ```LIMIT 1```.

Reply:
The `LIMIT 1` would be superfluous, queries with an aggregate function without a `GROUP BY` statement will always return a single line. Duly noted though.